airflow/operators/subdag_operator.py

```diff
@@ -46,15 +44,16 @@ class SubDagOperator(BaseOperator):
     @apply_defaults
     def __init__(
             self,
-            subdag,
-            executor=SequentialExecutor(),
+            subdag: DAG,
             *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.subdag = subdag
+        session = kwargs.pop('session')
```

Comment:
If we explicitly pop this from the `kwargs`, then we could also add it as an argument.

Reply:
Fixed.